internal/app/skuba/cluster/init.go

```diff
@@ -89,5 +91,7 @@ func NewInitCmd() *cobra.Command {
 	cmd.Flags().StringVar(&initOptions.CloudProvider, "cloud-provider", "", "Enable cloud provider integration with the chosen cloud. Valid values: openstack")
 	cmd.MarkFlagRequired("control-plane")
 
+	cmd.Flags().BoolVar(&initOptions.StrictCapDefaults, "strict-cap-defaults", false, "All the containers will start with CRI-O default capabilities")
```

Comment:
remove `cap` from this. while their product should run on ours, I don't think this setting should be named after it.

Reply:
that being said. I would do "strict by default" and say you turn this into `custom-ciro-config` I would then change the rest of this update to support that.

aka, by default it continues to work the same way it always has. If you want a less restrictive run you can run with `--custom-crio-config` (or something like that.)

Reply:
`cap` in that case stands for capabilities (not SUSE CAP) :P 

I would also prefer by default to do the default cri-o stuff, but the PM and PO said otherwise.

Reply:
> `cap` in that case stands for capabilities (not SUSE CAP) :P

:man_facepalming: i thought it stands for `SUSE CAP`... this is quite misleading, do you think we could call it `--strict-capability-defaults` ?

Reply:
Sure I can do that.